### ISSUE TITLE
conda -p name ... can't be activated (on windows?)

### ISSUE THREAD

**Issue by janschulz** [conda/conda#1664]
I created a new conda env in the current directory:

```
λ conda create -p mew_test python matplotlib seaborn ipython
[...]
Proceed ([y]/n)? y

Fetching packages ...
decorator-4.0. 100% |###############################| Time: 0:00:00   2.83 MB/s
numpy-1.9.3-py 100% |###############################| Time: 0:00:31 779.73 kB/s
pytz-2015.6-py 100% |###############################| Time: 0:00:01 157.85 kB/s
Extracting packages ...
[      COMPLETE      ]|##################################################| 100%
Linking packages ...
[      COMPLETE      ]|##################################################| 100%
#
# To activate this environment, use:
# > activate c:\data\phd\montecarlo-paper\mew_test
#
```

I then tried to run the supplied command, but it didn't work:

```
λ activate c:\data\phd\montecarlo-paper\mew_test
No environment named "c:\data\phd\montecarlo-paper\mew_test" exists in C:\portabel\miniconda\envs

λ activate mew_test
No environment named "mew_test" exists in C:\portabel\miniconda\envs
```

Not sure what's wrong here, but either the help text needs an update or the activate script.

**Comment by btlorch** [comment]
Got the same issue. Seems like the environment is created in my home directory in C:\Users\User1\.conda\envs but conda looks in the installation directory. First I tried to symlink the envs directory from my home directory into the installation directory, no success though.
I had installed Miniconda for all users on the computer. Uninstalled it and ticked to "Only for me" box and this way it is working.
